README.md

```diff
@@ -13,17 +13,40 @@ Check out the `examples` folder for detailed descriptions and code examples for:
 
 # Installation
 
-Phoenics can be installed via pip
+## Environment (Optional)
+
+It is suggested to have a fresh environment in order to avoid to have pip modules compatibility issues. This is optional but recommended.
+
+This can be done with Venv or Anaconda.
+
+### Venv
+```bash
+python3.6 -m venv phoenics-env
+source phoenics-env/bin/activate
 ```
-  apt-get install python-pip
-  pip install phoenics
+A `venv` folder will be created in the current directory and the virtual enviroment will be activated.
+
+### Anaconda
+```bash
+conda create --name phoenics-env python=3.6
+conda activate phoenics-env
 ```
 
-or by cloning this repository and building it from source
+## Phoenics as pip module
+
+The Phoenics module can be installed directly using the function `python-pip`.
+```bash
+apt-get install python-pip
+pip install phoenics
 ```
-  git clone https://github.com/chemos-inc/phoenics.git
-  cd phoenics
-  python setup.py install
+
+## Phoenics from source code
+
+Phoenics can also be installed from the source code. This way allows anyone to make and test changes on the code.
```

Comment:
Phoenics can also be installed from source

Reply:
Done